src/debug/daccess/daccess.cpp

```diff
@@ -7509,6 +7509,7 @@ STDAPI CLRDataAccessCreateInstance(ICLRDataTarget * pLegacyTarget,
 #ifdef __GNUC__
 __attribute__((used))
```

Comment:
Do we need this attribute when this is marked with `DLLEXPORT`?

Reply:
Nope, at least not by clang (i think true all supported CI matrix) and gcc (on ubuntu) - #removed